src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -1,3 +1,43 @@
-<p>
-  instructor-help-page works!
-</p>
+<main>
+    <h1 class="color-orange"> Help for Instructors</h1>
```

Comment:
Any reason why it is 4 spacing in this file instead of 2?

Reply:
No reason! I think my formatter messed up.. I have fixed it now.